build.sbt

```
@@ -184,6 +184,8 @@ lazy val V = new {
   val scala210 = "2.10.7"
   val scala211 = "2.11.12"
   val sbtScala = "2.12.10"
+  // TODO https://github.com/scalameta/metals/issues/2392
+  val legacy212 = "2.12.12"
```

```suggestion
  val ammonite212Version = "2.12.12"
```
We can keep it as is and when all is fixed change it to:
`  val ammonite212Version = scala212`

Since this happens frequently.

---

Ah, good idea 👍🏼 changed.